### DISCUSSION TITLE
How to make the visualization result of rosbag a line instead of points?

### DISCUSSION THREAD

**Discussion by 1045245078** [foxglove/community#912] [General]
![image](https://github.com/foxglove/community/assets/52687970/456e5739-10e5-4857-bfc4-80690ad5c76f)
I have already turned on the ‘show line’ setting, but by default it still displays points.

**Comment by defunctzombie** [comment]
Can you try the latest version at https://studio.foxglove.dev and see if you still experience this issue? We recently merged a change that should address this. If you are still not able to see lines - please report back maybe with more steps to reproduce.
